include/libopencm3/nrf/common/i2c.h

```diff
@@ -75,9 +75,16 @@
 /* Register Contents */
 
 /** @addtogroup i2c_shorts I2C event -> task shortcuts
+ * Shortcut, which is activated causes that if certain event
+ * occurrs, chosen task is automatically acitvated by the
+ * hardware.
```

Comment:
this has typos and reads strangely?  does this make sense automatically to familiar NRF users?

Reply:
I should not write documentation before I drink morning coffee